src/primitives/transaction.h

```diff
@@ -8,6 +8,7 @@
 
 #include <stdint.h>
 #include <amount.h>
+#include <crypto/siphash.h>
```

Comment:
2ba96b0438fe734b788e56ba881303c84a3f249d
Why this header added?

Reply:
Oops, that's from a previous iteration of this PR. Removed.